server/src/test/java/org/cloudfoundry/identity/uaa/scim/jdbc/JdbcScimUserProvisioningTests.java

```diff
@@ -1048,26 +1053,66 @@ void updateLastLogonTime() {
 
     @Test
     void cannotCreateMaxUserLimit() {
-        IdentityZoneHolder.get().getConfig().getUserConfig().setMaxUsers(10);
         ScimUser scimUser = new ScimUser("user-id-1", "user1@example.com", "User", "Example");
         ScimUser.Email email = new ScimUser.Email();
         email.setValue("user@example.com");
         scimUser.setEmails(Collections.singletonList(email));
-        scimUser.setPassword("password");
+        scimUser.setPassword(randomString());
         scimUser.setOrigin(OriginKeys.UAA);
-        for (int i = 2; i < 12; i++) {
+        try {
+            idzManager.getCurrentIdentityZone().getConfig().getUserConfig().setMaxUsers(10);
+            for (int i = 1; i < 12; i++) {
+                scimUser.setId("user-id-" + i);
+                scimUser.setUserName("user" +i+ "@example.com");
+                scimUser.setPassword(randomString());
+                scimUser = jdbcScimUserProvisioning.create(scimUser, IdentityZoneHolder.getCurrentZoneId());
+            }
+        } catch (InvalidScimResourceException e) {
```

Comment:
Either use `assertThrows` or add `fail` before this line

Reply:
fixed